parl/remote/cpu_resource.py

```diff
@@ -0,0 +1,80 @@
+#   Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import defaultdict
+from parl.remote.message import InitializedCpu
+
+
+class CpuResource(object):
+    """The cpu center deals with everything related to allocation for CPUs.
```

Comment:
Record CPU information in each worker and respond to resource allocation of CPUs.